sdk/storage/azure_storage_blob/tests/test_blob_client.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// use azure_core_test::recorded;
+// use azure_identity::DefaultAzureCredentialBuilder;
+// use azure_storage_blob::{BlobBlobClientGetPropertiesOptions, BlobClient, BlobClientOptions};
+// use std::{env, error::Error};
+
+// #[recorded::test(live)]
+// async fn test_get_blob_properties() -> Result<(), Box<dyn Error>> {
```

Comment:
Why is all this commented out? If you're not ready to record, fine. But you've already marked them as live-only tests anyway so that's not a concern. We still want to build them to make sure they work.

Reply:
> I have commented out the testcases since they currently rely on static resources and have a hard dependency on some NYI functionality.

So I can see your point that they are innocuous because they are live-only, but will leave them commented out since they don't really make any sense until the next PR which will have some NYI APIs.